web-animations-1/Overview.bs

```diff
@@ -4505,12 +4512,20 @@ dictionary OptionalEffectTiming {
     the number of milliseconds from the [=start time=] of the associated
     <a>animation</a> to the start of the <a>active interval</a>.
 
+    If the <a>iteration duration</a> resolves to a percentage, the start
```

Comment:
(Nit: What does "resolves to" mean here?)

Reply:
I switched the wording on this to is treated as which matches the wording for the iteration duration when "auto" is specified.